templates/strings/objc-h.stencil

```diff
@@ -0,0 +1,50 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+{% macro parametersBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    ({% call paramTranslate type %})p{{ forloop.counter }}{% if not forloop.last %} and:{% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro argumentsBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    p{{forloop.counter}}{% if not forloop.last %}, {% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro paramTranslate swiftType %}
+  {% if swiftType == "Int" %}
+    NSInteger
+  {% elif swiftType == "String" %}
+    NSString*
+  {% else %} 
+    NSObject*
```

Comment:
imho `id` would be more accurate

Reply:
Are you referring to line 22 (`NSString*`) or line 24 (`NSObject*`)?

If the latter, then I changed the output for all the things I don't handle to `FIX_OBJC_STRING_DOT_STENCIL_PLEASE`.  That should make it easier for the next guy to find where the fix goes.